tests/integration/models/reddit/test_comment.py

```diff
@@ -153,9 +153,11 @@ def test_parent__submission(self):
         assert parent.fullname == comment.parent_id
 
     def test_refresh(self):
+        comment = Comment(self.reddit, "d81vwef")
         with self.recorder.use_cassette("TestComment.test_refresh"):
-            comment = Comment(self.reddit, "d81vwef").refresh()
-        assert len(comment.replies) > 0
+            assert len(comment.replies) == 0
```

Comment:
This assertion can be outside of the cassette, right?

Reply:
No.

Reply:
How does this trigger a fetch? And if it does, it's the wrong value.

Reply:
The `.replies` property calls [`.submission`][1] which in turn calls [`_extract_submission_id()`][2] which triggers a fetch because [`link_id`][3] is read.

[1]: https://github.com/praw-dev/praw/blob/master/praw/models/reddit/comment.py#L112
[2]: https://github.com/praw-dev/praw/blob/master/praw/models/reddit/comment.py#L120
[3]: https://github.com/praw-dev/praw/blob/master/praw/models/reddit/comment.py#L196

> And if it does, it's the wrong value.

I’m not sure what you mean by this?